inc/FileSystemPolicy.hpp

```diff
@@ -9,6 +9,7 @@
 #include <iostream>
 #include <sys/types.h>
 #include <dirent.h>
+#include <unistd.h>
```

Comment:
This is new, I think it can be removed? Maybe for forbidden c functions

Reply:
Removed